src/main/java/com/softserve/demo/controller/ProvidersController.java

```diff
@@ -79,9 +83,21 @@ public void uploadImage(
 
     @GetMapping("find-all/status")
     @PreAuthorize("hasAnyRole('ROLE_ADMIN')")
-    public Page<?> getServiceProviderByStatus(@PageableDefault Pageable pageable,
+    public Page<?> getServiceProviderByStatus(@RequestParam(defaultValue = "0") int page,
+                                              @RequestParam int numberOfProvidersOnPage,
                                               @RequestParam(defaultValue = "NOTAPPROVED") String status) {
-        return providersService.getServiceProvidersByStatus(pageable, ProviderStatus.valueOf(status));
+        return providersService.getServiceProvidersByStatus(page,numberOfProvidersOnPage, ProviderStatus.valueOf(status));
+    }
+
+    @GetMapping("find-all/searchByName")
+    @PreAuthorize("hasAnyRole('ADMIN')")
+    @ResponseStatus(HttpStatus.OK)
+    public Page<ProviderInfoDTO> getServiceProviderByName(@RequestParam(defaultValue = "0") int page,
+                                                          @RequestParam int numberOfProvidersOnPage,
+                                                          @RequestParam(defaultValue = "NOTAPPROVED") String status,
```

Comment:
Is there a way we can receive empty value for status? My concern is that when we change Enum with statuses we can easily forget to change this one. May be we can omit default value for status?